src/algorithms/explicit_rk.jl

```diff
@@ -0,0 +1,50 @@
+function explicit_rk_docstring(description::String)
+    start_docstring = """
+       ```julia
+       $FUNCTIONNAME(; stage_limiter! = OrdinaryDiffEq.trivial_limiter!,
```

Comment:
You're literally interpolating the `string()` value of the `FUNCTIONNAME` object into the string here, hence why it doesn't work. DocStringExtensions takes advantage of the fact that string interpolation in macro calls / docstrings is special, which is how it works in `@doc "$FUNCTIONNAME ..." foo` situations.

I don't see any way to make this work with DocStringExtensions, but you could, of course, work around it by passing the function name as an extra parameter to `explicit_rk_docstring`. I would suggest opening an issue at DocStringExtensions though, since this is an interesting case.

Reply:
Looks kind of like what [`@template`](https://docstringextensions.juliadocs.org/latest/#DocStringExtensions.@template) is for perhaps. If that's stretching what the template macro can actually handle currently it may be possible to improve it's features to handle this case better.